Tests/Marketplace/marketplace_services.py

```diff
@@ -1780,7 +1786,8 @@ def filter_entries_by_display_name(release_notes: dict, display_names: list, rn_
         for display_name, rn_entry in release_notes[rn_header].items():
 
             logging.debug(f"Searching display name '{display_name}' in '{display_names}'.")
-            if display_name != '[special_msg]' and display_name.replace("New: ", "") not in display_names:
+            if display_name != '[special_msg]' and display_name.replace("New: ", "") not in display_names \
+                and not get_id_set_entity_by_display_name(display_name.replace("New: ", ""), pack_folder, id_set):
```

Comment:
Why do we need the second condition? I think it is covered in the last condition